starlight/static/css/main.css

```diff
@@ -0,0 +1,28 @@
+@import url('https://fonts.googleapis.com/css?family=Proxima+Nova');
+
+body {
+  /*background: url('/static/images/starlight-background.png');*/
+  background-color: lightgray;
+  background-size: 100% auto;
+  font-family: 'proxima-nova';
+}
+body > .grid {
+  height: 100%;
+}
+.image {
+  margin-top: -100px;
```

Comment:
If you do this, every single image on the page could appear 100px above where it should... is this what you want? :)